.github/mypy_check.sh

```diff
@@ -0,0 +1,5 @@
+if [ "$(basename "$(pwd)")" = ".github" ]; then
+    cd ..
+fi
+
+xargs mypy --strict --follow-imports=silent --no-warn-unused-ignore --no-warn-return-any --install-types --non-interactive typings < .github/mypy_files.txt
```

Comment:
If I do `import schema` in an options.py file, I get this:

`worlds\witness\options.py:3: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker  [import-untyped]`

That probably needs to be supressed? Not sure

Reply:
We could add `--disable-error-code=import-untyped`
But another solution, that is more type safe, but also takes more work, is to make our own type stubs for libraries we use that don't have type information.
That's what the `typings` directory is for in Archipelago. I've been maintaining some `kivy` type stubs there for a while.